### PR TITLE
Ports: Add imgcat port

### PR DESCRIPTION
**Pull request by panos**
- [imgcat's GitHub repository](https://github.com/eddieantonio/imgcat)

### CODE REVIEW

File: Ports/imgcat/package.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=imgcat
+version=2.5.0
+useconfigure=false
+workdir=imgcat-2.5.0
```
Comment (linusg) [inline Ports/imgcat/package.sh:5]:
```suggestion
```

File: Ports/imgcat/package.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+files="https://github.com/eddieantonio/imgcat/releases/download/v${version}/imgcat-${version}.tar.gz imgcat-v2.5.0.tar.gz 1b5d45ccafc6fbb7a7ee685d4a5110101418d48b"
```
Comment (linusg) [inline Ports/imgcat/package.sh:6]:
```suggestion
files="https://github.com/eddieantonio/imgcat/releases/download/v${version}/imgcat-${version}.tar.gz imgcat-v${version}.tar.gz 1b5d45ccafc6fbb7a7ee685d4a5110101418d48b"
```

**Review by linusg — COMMENTED** [review]
Please also add to AvailablePorts.md :)

**Comment by panos** [comment]
Sorry, I accidentally deleted the file and tried to recreate it hastily because I've gotta run for now, I'll quickly squash the changes.

**Comment by gunnarbeutner** [comment]
Out of curiosity, did this actually build successfully for you?

**Comment by panos** [comment]
> Out of curiosity, did this actually build successfully for you?

Yeah, I didn't seem to have an issue with it. I'll try again just to be sure.

**Comment by panos** [comment]
Yeah... there's definitely something wrong here. I'll take a look.

**Comment by panos** [comment]
I'm facing hell.

I'll revert the change.

**Pull request merged**